_posts/blog/2019-05-21-Monica's-Introduction

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Google Summer of Code - Monica's Introduction
+categories: blog
+excerpt: Monica, who will be joining the team as a part of Google Summer of Code, introduces herself!
+image:
+  feature:
+link:
+date: 2019-05-20
+modified:
+share: true
+author: monicayao
+---
+
+## About Me
+
+My name is Monica and I am currently an undergraduate student at Harvey Mudd College (located in the beautiful Southern California). 
+I am joining the Whitaker Lab as a part of the Google Summer of Code program. My interests for computer science are very recent - 
+a year ago, I had wanted to be a surgeon! I have discovered so many useful computer science applications for the medical field. 
+Through working with the Tedana library, I hope to assist in denoising fMRI data, and dive deeper into the real life applications of CS!
```

Comment:
I think I’ve always written tedana in all lowercase, like the package name. What do you think?